asserts/store_asserts.go

```diff
@@ -0,0 +1,94 @@
+package asserts
+
+import (
+	"fmt"
+	"net/url"
+)
+
+// EnterpriseStore holds an enterprise-store assertion, defining the
+// configuration needed to connect a device to the enterprise store.
+type EnterpriseStore struct {
```

Comment:
Oh, I totally agree! I really don't like this name for the assertion, but I assumed the name had already been decided. I will gladly change it!

I'd wondered about `alt-store` (short for "alternative", in case it's not clear), `device-store` (but not sure I like the "device" connotation much), or just `store`. I'll leave it until my tomorrow PM before I change things in case you prefer any of my suggestions, otherwise I'll go with `custom-store`.